package-manager-configs/nx-staging/rpm/sonatype-community.repo

```diff
@@ -1,7 +1,7 @@
 [sonatype-community]
 name=Sonatype Community
-baseurl=https://nx-staging.sonatype.com/repository/community-yum-hosted/
+baseurl=https://repo.sonatype.com/repository/community-yum-hosted-staging/
```

Comment:
I don't think these links should change because they refer to the old `nx-staging`. Similarly, the repo name (`community-hosted`) is the same on the staging server as the production server.

Or is a change a-foot to the nx-staging repo names? If so, there's lot's of other changes that will need to happen (e.g. CI config, etc).

Reply:
> I don't think these links should change because they refer to the old `nx-staging`. Similarly, the repo name (`community-hosted`) is the same on the staging server as the production server.
> 
> Or is a change a-foot to the nx-staging repo names? If so, there's lot's of other changes that will need to happen (e.g. CI config, etc).

Yeah the nx-staging host needs to be shutdown. This was to move to the host on a staging specific repo.

Reply:
Would it be easier just to "nuke" all the "staging" flavors? I don't see staging repos getting much use (and I basically don't use them at all since moving to the production repo).